src/components/general/Navbar/Notifica/Notifications/items/EntryCommentItem/index.js

```diff
@@ -102,6 +102,12 @@ export default class EntryCommentItem extends React.PureComponent {
                 'entryCommentReplyAdd',
                 stringParams,
             );
+        } else if (notificationType === 'entry_comment_assignee_change') {
+            notificationText = _ts(
+                'entryComments.notifications',
+                'entryCommentAdd',
```

Comment:
Should this not have a different messag than entryCommentAdd?

Reply:
No it's fine, it will follow the same syntax.